carbon_cassandra_plugin/carbon_cassandra_db.py

```diff
@@ -364,7 +365,13 @@ def _metric_lookup(query_split, query_re, level, child_nodes):
       # Also, "root" will not be a literal match against most metrics using RE.
       if query == "root" or not _query_contains_wildcards(query):
         child_nodes = _get(query)
+      elif query[0] == '*':
+        # We don't prepend our metrics with "root". The extra look up works around that.
+        for key, col, value in _get("root"):
```

Comment:
Why can't `_metric_lookup` handle this case? Shouldn't it not matter that the first component of the query is `*` since it should do this same process anyway?